cluster/addons/podsecuritypolicies/privileged.yaml

```diff
@@ -1,23 +1,24 @@
 apiVersion: extensions/v1beta1
 kind: PodSecurityPolicy
 metadata:
-  annotations:
-    kubernetes.io/description: 'privileged allows access to all privileged and host
-      features and the ability to run as any user, any group, any fsGroup, and with
-      any SELinux context.'
-  creationTimestamp: 2016-05-06T19:28:58Z
   name: privileged
+  annotations:
+    kubernetes.io/description: 'privileged allows full unrestricted access to
+      pod features, as if the PodSecurityPolicy controller was not enabled.'
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
```

Comment:
I feel that we miss `apparmor.security.beta.kubernetes.io/allowedProfileNames` here.

Reply:
Unlike with seccomp, the default for AppArmor is allow-all. Going forward, that needs to be the default value to avoid breaking existing profiles.